LayoutTests/fast/css/caret-color-auto.html

```diff
@@ -24,7 +24,7 @@
 </style>
 </head>
 <body>
-<p>Tests that "caret-color: auto" behaves identical to omitting property caret-color.</p>
+<p>Tests that "caret-color: auto" causes the caret to take the "color" property of its parent, if applicable.</p>
```

Comment:
It's not the color property of the parent, it's the color property of the element itself afaik